app/src/main/java/net/squanchy/proximity/BluetoothModule.java

```diff
@@ -0,0 +1,17 @@
+package net.squanchy.proximity;
+
+import android.app.Activity;
+import android.bluetooth.BluetoothManager;
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class BluetoothModule {
+
+    @Provides
+    BluetoothManager bluetoothManager(Activity activity) {
```

Comment:
Good luck injecting a `Context`. Dagger will explode because there's different kinds of `Context`, `Application` and `Activity`, and it doesn't know which one to use. This could, I suppose, use an `Application` instead. But no, you cannot have a `@Provide Context` if you need to also distinguish between `Activity` and `Application` in other modules (which is our case).